.github/actions/setup-macos-sdk.sh

```diff
@@ -3,13 +3,15 @@
 XCODE_MACOS_PLATFORM_PATH="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform"
 XCODE_MACOS_SDK_PATH="${XCODE_MACOS_PLATFORM_PATH}/Developer/SDKs"
 
-SDK_VER=10.14
+if [[ -z ${MACOS_SDK_VER+x} ]]; then
+    MACOS_SDK_VER=10.14
+fi
```

Comment:
Am I reading this correctly that this is stating that if the string MACOS_SDK_VER is empty then define it.  What is the +x for?

Reply:
The +x is some bash parameter substitution; doesn't look like it is needed here and works fine without it.

The default macOS SDK version we're targeting is 10.14; all our uses of this script should be defining MACOS_SDK_VER already (to 10.14)